pyctcdecode/decoder.py

```diff
@@ -667,35 +683,33 @@ def decode_batch(
 
 def build_ctcdecoder(
     labels: List[str],
-    kenlm_model: Optional[kenlm.Model] = None,
-    unigrams: Optional[Iterable[str]] = None,
+    kenlm_model_path: Optional[str] = None,
+    unigrams: Optional[Collection[str]] = None,
     alpha: float = DEFAULT_ALPHA,
     beta: float = DEFAULT_BETA,
     unk_score_offset: float = DEFAULT_UNK_LOGP_OFFSET,
     lm_score_boundary: bool = DEFAULT_SCORE_LM_BOUNDARY,
-    ctc_token_idx: Optional[int] = None,
-    is_bpe: bool = False,
 ) -> BeamSearchDecoderCTC:
     """Build a BeamSearchDecoderCTC instance with main functionality.
 
     Args:
         labels: class containing the labels for input logit matrices
-        kenlm_model: instance of kenlm n-gram language model `kenlm.Model`
-        unigrams: list of known word unigrams
+        kenlm_model_path: path to kenlm n-gram language model
+        unigrams: list of known word unigrams, this will greatly improve accuracy
```

Comment:
editorializing :)

Reply:
If we want to gloss this we should probably just have a full paragraph after the arg listing

`

Reply:
yeah, but since it's docstring of the convenience function i suppose it's fine to be a bit more colloquial?

Reply:
I would at least want to gloss how unigrams interacts with the LM.  If the user provides an LM but not unigrams, we take the unigrams from the LM anyway.  So if the user sees "this will greatly improve accuracy" and takes the unigram list from their LM, they'll just be reduplicating `load_unigram_set_from_arpa` for no benefit, I think.  And if the user provides both an LM and unigrams, we ignore everything in the LM-- is that what we want? (asking)

Reply:
hm, yes we do want that, but i see how it's not obvious :) will maybe add as logging